src/Engine/Component/CameraComponents/CameraComponent.cpp

```diff
@@ -53,11 +76,13 @@ void CameraComponent::init() {
     Scene::addReceiver<SpatialRotatedMessage>(&gameObject(), spatRotationCallback);
     Scene::addReceiver<CollisionAdjustMessage>(&gameObject(), spatTransformCallback); // necessary as collision sets position silently
 
-    auto windowSizeCallback([&] (const Message & msg_) {
-        m_projMatValid = false;
-        m_frustumValid = false;
-    });
-    Scene::addReceiver<WindowFrameSizeMessage>(nullptr, windowSizeCallback);
+    if (!m_isOrtho) {
```

Comment:
Should this if-statement be here?

Reply:
This is fine for now. Prob the camera component shouldn't interface with Window at all. Should be the responsibility of something else to update it when the window changes.